### PR TITLE
Feature 52 proposals content type and views

### PR DESCRIPTION
**Pull request by alvincases**
Proposal view when there is no submission yet.
      Desktop:
image1
      Mobile:
image3

Proposals view
    Desktop:
image12
    Mobile: 
image13

Proposal submission form, all pages
     Desktop:
image14
image6
image9

     Mobile:
image4
image5
image8

Thank you page after submission
     Desktop:
image10
     Mobile:
image11

### CODE REVIEW

File: static/sass/partials/_content.scss [hunk 1]
```diff
@@ -45,3 +45,155 @@
     }
   }
 }
+
+.view-footer a.submit-proposal-btn {
+  background: #cd853f;
```
Comment (Luukyb) [inline static/sass/partials/_content.scss]: Could we use or re-use a variable?

File: static/sass/partials/_content.scss [hunk 2]
```diff
@@ -45,3 +45,155 @@
...
+  padding: 8px;
+  color: $primary-bg;      
+}
+.view-footer a.submit-proposal-btn:hover {
+  background-color: rgba(205, 133, 63, 0.9);
+}
+
+#edit-actions {
+  .next-btn {
+    background: #cd853f;
```
Comment (Luukyb) [inline static/sass/partials/_content.scss]: Could we use or re-use a variable?

File: static/sass/partials/_content.scss [hunk 3]
```diff
@@ -45,3 +45,155 @@
...
+    width: 100%;
+    margin-bottom: 50px;
+    margin-top: 25px; 
+    color: $primary-bg;  
+  }
+  .next-btn:hover {
+    background-color: rgba(205, 133, 63, 0.9);
+  }
+  .previous-btn {
+    background: #cd853f;
```
Comment (Luukyb) [inline static/sass/partials/_content.scss]: Could we use or re-use a variable?

File: static/sass/partials/_content.scss [hunk 4]
```diff
@@ -45,3 +45,155 @@
...
+    padding: 6px 25px;
+    width: 100%;
+    margin-top: 25px; 
+    color: $primary-bg;  
+  }
+  .previous-btn:hover {
+    background-color: rgba(205, 133, 63, 0.9);
+  }
+  .submit-btn {
+    background: #cd853f;
```
Comment (Luukyb) [inline static/sass/partials/_content.scss]: Could we use or re-use a variable?

File: static/sass/partials/_content.scss [hunk 5]
```diff
@@ -45,3 +45,155 @@
...
+  margin-bottom: 50px;
+  text-align: center;
+  padding-top: 30px;
+  a.submit-proposal-btn {
+    padding: 16px 40px;
+  }
+}
+
+.view-proposals .views-row {
+  background-color: #f5f5f5;
```
Comment (Luukyb) [inline static/sass/partials/_content.scss]: Could we use or re-use a variable?

File: static/sass/partials/_content.scss [hunk 6]
```diff
@@ -45,3 +45,155 @@
...
+    text-align: center;
+    font-size: 20px;
+    padding-top: 50px;
+  }
+  
+  .see-all-proposal-btn {
+    padding-top: 40px;
+    margin-bottom: 40px;
+    a {
+      background: #cd853f;
```
Comment (Luukyb) [inline static/sass/partials/_content.scss]: Could we use or re-use a variable?

File: static/sass/partials/_content.scss [hunk 7]
```diff
@@ -45,3 +45,155 @@
...
+      color: $primary-bg;  
+    }
+    a:hover {
+      background-color: rgba(205, 133, 63, 0.9);
+    }
+  }
+}
+
+#webform-submission-session-proposals-add-form {
+
```
Comment (Luukyb) [inline static/sass/partials/_content.scss]: Extra line.

File: static/sass/partials/_content.scss (hunk 1 above)
Comment (alvincases) [reply static/sass/partials/_content.scss]: Done

File: static/sass/partials/_content.scss (hunk 2 above)
Comment (alvincases) [reply static/sass/partials/_content.scss]: Done

File: static/sass/partials/_content.scss (hunk 3 above)
Comment (alvincases) [reply static/sass/partials/_content.scss]: Done

File: static/sass/partials/_content.scss (hunk 4 above)
Comment (alvincases) [reply static/sass/partials/_content.scss]: Done

File: static/sass/partials/_content.scss (hunk 5 above)
Comment (alvincases) [reply static/sass/partials/_content.scss]: Done

File: static/sass/partials/_content.scss (hunk 6 above)
Comment (alvincases) [reply static/sass/partials/_content.scss]: Done

File: static/sass/partials/_content.scss (hunk 7 above)
Comment (alvincases) [reply static/sass/partials/_content.scss]: Done

**Pull request merged**